linker-external.go

```diff
@@ -14,8 +14,12 @@ import (
 //
 // This version always runs the linker as an external command.
 func Link(linker string, flags ...string) error {
-	cmd := exec.Command(linker, flags...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-	return cmd.Run()
+	if cmdNames, ok := commands[linker]; ok {
+		return execCommand(cmdNames, flags...)
+	} else {
```

Comment:
This else is superfluous

Reply:
It is not when linking with the system compiler.

https://github.com/tinygo-org/tinygo/blob/fa5df4f524c3b4f15360c2da964932692e3ab4af/target.go#L215

Reply:
I'm confused. Whats wrong with this?
```suggestion
if cmdNames, ok := commands[linker]; ok {
	return execCommand(cmdNames, flags...)
}
cmd := exec.Command(linker, flags...)
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
return cmd.Run()
```

Reply:
Oh, a formatting issue (not a logic issue). Yes, I'll apply this.